components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1759,9 +1798,13 @@ bool AdsServiceImpl::ShouldShowNotifications() {
 
 void AdsServiceImpl::CloseNotification(
     const std::string& uuid) {
-  std::unique_ptr<PlatformBridge>
-    platform_bridge = std::make_unique<PlatformBridge>(profile_);
-  platform_bridge->Close(profile_, uuid);
+  if (brave::IsDevOrCanaryBuild()) {
+    std::unique_ptr<PlatformBridge>
+      platform_bridge = std::make_unique<PlatformBridge>(profile_);
+    platform_bridge->Close(profile_, uuid);
+  } else {
+    display_service_->Close(NotificationHandler::Type::BRAVE_ADS, uuid);
```

Comment:
Not all changes have been reverted, see https://github.com/brave/brave-core/pull/6728

Reply:
I will need to dive a bit deeper on this, will take a look.

Reply:
Agreed with @yachtcaptain23 this logic is required to limit the amount of ad notifications that appear in the system tray on Android